### DISCUSSION TITLE
Random CrashLoopBackOff in replicas

### DISCUSSION THREAD

**Discussion by netthier** [cloudnative-pg/cloudnative-pg#957] [Q&A] [answered]
Hello, I have a Cluster with 3 replicas. From time to time, on of them randomly starts entering a CrashLoopBackOff with these logs: https://hastebin.com/iwotazewil.json
Anyone know what might be going on there?

EDIT: Just noticed that the S3 backup had some stale data from a previous database left, could that have caused it?

**Comment by gbartolini** [comment, marked as answer]
It is a desired behaviour.

Quoting from https://cloudnative-pg.io/documentation/current/backup_recovery/#restoring-into-a-cluster-with-a-backup-section

> The operator includes a safety check to ensure a cluster will not overwrite a storage bucket that contained information. A cluster that would overwrite existing storage will remain in state Setting up primary with Pods in an Error state. The pod logs will show: ERROR: WAL archive check failed for server recoveredCluster: Expected empty archive

We need to make sure that you are not using a backup object store that's been used in the past.

**Reply by netthier** [reply]
That line doesn't show up in the logs though, also I'd expect the backup to fail, not the whole database.
